src/geo_types_from_wkt.rs

```diff
@@ -0,0 +1,884 @@
+//! This module provides conversions between WKT primitives and [`geo_types`] primitives.
+//!
+//! See the [`std::convert::From`] and [`std::convert::TryFrom`] impls on individual [`crate::types`] and [`Wkt`] for details.
+//!
+//!
+//!
+//!
+//!
+// Copyright 2014-2018 The GeoRust Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
This was cut and pasted from `conversion.rs`, just because I think this is a clearer name and has better symmetry with the new `geo_types_to_wkt`module.